public/systems_monitoring_graphs.js

```diff
@@ -0,0 +1,1087 @@
+                .attr("transform", "rotate(-65)");
+
+            // Add X axis label
+            svg.append("text")
+                .attr("text-anchor", "end")
+                .attr(
+                    "transform",
+                    "translate(" + Width / 2 + " ," + (Height + Margin.top + 50) + ")"
+                )
+                .style("text-anchor", "middle")
+                .text("Date (dd:hh:m)");
+
+            // Add Y axis label
+            svg.append("text")
+                .attr("transform", "rotate(-90)")
+                .attr("y", 0 - Margin.left)
+                .attr("x", 0 - Height / 2)
+                .attr("dy", "1em")
+                .style("text-anchor", "middle")
+                .text("Disc Usage (GB)");
+                
+            let yLimit = data[0].disk_usage.total
+            // Add Y axis
+            let decimalFormatter = d3.format(".2s");
+            let y = d3.scaleLinear()
+                // Add 100 to yLimit to increase brushing area
+                .domain([0, yLimit + 100])
+                .range([ Height, 0 ]);
+                svg.append("g")
+                .call(d3.axisLeft(y).ticks(5).tickFormat((d) => decimalFormatter(d).replace('G', 'GB')))      
+
+            // Add the Y gridlines
+            svg.append("g")			
+                .attr("class", "diskGrid")
+                .call(d3.axisLeft(y)
+                    .tickSize(-Width)
+                    .tickFormat("")
+                )
+
+            // Get time when the system stopped & when it was restarted
+            let stop = [], start = [], currentTime = new Date();
+            data.forEach((d, i, n)=> {
+                if (i == (n.length-1)) {
+                    let difference_ms = (currentTime.getTime() - d.datetime.getTime()) / 60000,
+                    difference_minutes = Math.floor(difference_ms % 60);
+                    if (difference_minutes > 10) {
+                        d.disk_usage.used = 0;
+                        stop.push(d.datetime)
+                    }
+                } else if (i>0) {
+                    let difference_ms = (d.datetime.getTime() - n[i-1].datetime.getTime()) / 60000,
+                    difference_minutes = Math.floor(difference_ms % 60);
+                    if (difference_minutes > 10) {
+                        n[i-1].disk_usage.used = 0;
+                        d.disk_usage.used = 0;
+                        stop.push(n[i-1].datetime)
+                        start.push(d.datetime)
+                    } 
+                }
+            })
+            
+            // Add a clipPath: everything out of this area won't be drawn.
+            svg.append("defs").append("svg:clipPath")
+                .attr("id", "clip-lines")
+                .append("svg:rect")
+                .attr("width", Width )
+                .attr("height", Height )
+                .attr("x", 1)
+                .attr("y", 0);
+
+            let hitchLines = svg.append('g')
```

Comment:
Slight preference for outageLines, because I think that's a little more descriptive of the type of hitch we've encountered

Reply:
Updated thanks.